src/swap/components/SwapSettingsSlippageToggle.tsx

```diff
@@ -0,0 +1,36 @@
+import { useState } from 'react';
+import { cn } from '../../styles/theme';
+import type { SwapSettingsSlippageToggleReact } from '../types';
+
+export function SwapSettingsSlippageToggle({
+  className,
+}: SwapSettingsSlippageToggleReact) {
+  const [slippageMode, setSlippageMode] = useState<'Auto' | 'Custom'>('Auto');
+
+  return (
+    <div
+      className={cn(
+        'flex items-center gap-2 dark:border-gray-700 dark:bg-gray-950',
+        className,
+      )}
+    >
+      <div className="flex flex-1 rounded-xl border border-gray-300 bg-gray-100 p-1 dark:border-gray-700 dark:bg-gray-950">
+        {['Auto', 'Custom'].map((option) => (
```

Comment:
I think if we abstract this in his own component, will make this code way easier to use.

Reply:
This should be cleaner when I refactor to a boolean. Can abstract after if it will improve the experience

Reply:
This entire component consists of the Auto and Custom buttons